mmpose/datasets/datasets/top_down/topdown_crowdpose_dataset.py

```diff
@@ -179,22 +179,8 @@ def _load_coco_keypoint_annotation_kernal(self, index):
                 joints_3d_visible[ipt, 1] = t_vis
                 joints_3d_visible[ipt, 2] = 0
 
-            interference_num_joints = len(obj['keypoints']) // 3 - num_joints
-            interference_joints_3d = np.zeros((interference_num_joints, 3),
-                                              dtype=np.float)
-            interference_joints_3d_vis = np.zeros((interference_num_joints, 3),
-                                                  dtype=np.float)
-            for ipt in range(interference_num_joints):
-                interference_joints_3d[
-                    ipt, 0] = obj['keypoints'][(num_joints + ipt) * 3 + 0]
-                interference_joints_3d[
-                    ipt, 1] = obj['keypoints'][(num_joints + ipt) * 3 + 1]
-                interference_joints_3d[ipt, 2] = \
-                    obj['keypoints'][(num_joints + ipt) * 3 + 2]
-
-                interference_joints_3d_vis[ipt, 0] = 1
-                interference_joints_3d_vis[ipt, 1] = 1
-                interference_joints_3d_vis[ipt, 2] = 0
+            interference_joints_3d = np.array(obj['keypoints']).reshape(
```

Comment:
forgot the interference_joints_3d_vis?

Reply:
I deleted it because I found it unnecessary